src/wallet/wallet.cpp

```diff
@@ -280,17 +280,17 @@ bool CWallet::LoadWatchOnly(const CScript &dest)
 bool CWallet::Unlock(const SecureString& strWalletPassphrase)
 {
     CCrypter crypter;
-    CKeyingMaterial vMasterKey;
+    CKeyingMaterial _vMasterKey;
```

Comment:
So I believe changing the name of CCryptoKeyStore  master key will, I think change only four lines. Might be good to get an opinion from someone who's worked more on this code?  @TheBlueMatt @pstratem